packages/@react-aria/focus/stories/FocusScope.stories.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {boolean, withKnobs} from '@storybook/addon-knobs';
+import {FocusScope} from '../';
+import React, {ReactNode, useState} from 'react';
+import ReactDOM from 'react-dom';
+import {storiesOf} from '@storybook/react';
+
+const dialogsRoot = 'dialogsRoot';
+
+storiesOf('FocusScope', module)
```

Comment:
I think we're moving towards Component Story Format (CSF) for stories.

Reply:
https://github.com/adobe/react-spectrum/pull/2139/commits/9ad5946a1c55da57f059a2b9d0134c61d4ab0b3d